dotnet-interactive.Tests/JupyterInstallCommandTests.cs

```diff
@@ -4,12 +4,29 @@
 using System.CommandLine.IO;
 using System.Threading.Tasks;
 using FluentAssertions;
+using Microsoft.DotNet.Interactive.App.CommandLine;
 using Xunit;
 
 namespace Microsoft.DotNet.Interactive.App.Tests
 {
     public class JupyterInstallCommandTests
     {
+        [Fact]
+        public async Task Appends_http_port_range_arguments()
+        {
+            var console = new TestConsole();
+            var kernelSpec = new  InMemoryJupyterKernelSpec(true, null);
+            var jupyterCommandLine = new JupyterInstallCommand(console, kernelSpec , new PortRange{ Start = 100, End = 400});
+
+            await jupyterCommandLine.InvokeAsync();
+
+            foreach (var installedKernelSpec in kernelSpec.InstalledKernelSpecs)
```

Comment:
This test will pass if the collection is empty. It's safer to do something like `kernelSpec.InstalledKernelSpecs.Should().All...`